src/core/core/app_manager.cpp

```diff
@@ -247,7 +243,7 @@ namespace dsn
                 app = it->second.get();
             }
             else
-                return false;
+                return nullptr;
```

Comment:
Sorry I didn't pay much attention on this, but the function prototype is "rpc_handler_info* app_manager::rpc_unregister_handler(dsn_gpid gpid, dsn_task_code_t rpc_code)", which is not "bool".